styles/email/_email-extend-inline.scss

```diff
@@ -0,0 +1,6 @@
+//
+//  Email extend styles
+//  _____________________________________________
+
+//  This file should be used for theme-specific styles for emails that extend or modify the styles in email.scss
+//  This will allow you to edit email styles without copying and editing the email.scss file
```

Comment:
Let's remove this file, same as `email-extend`. Stylesheets are not designed to be extended, b/c it will just generate unused code, due to overwrites.

Reply:
In this case I would not agree. This doesn't forbid user to edit base files. However the base files are really minimum of styles for web clients and I think are good to work on top of it.

Reply:
I think we can use those files, maybe with "custom", not "extend" name, what do you think? And IMO, it should be information about styling emails in some README, to explain how to use it without generating so much of useless code.

Reply:
Again as sass-blank should be similar to less version I made it in convention like it is in less. 
The problem here was we can't easily make it like in less version where content of ***email-base.less*** is splited into ***email.less*** and ***email-inline.less***. The content that is inside `.email-non-inline() {...}` and inside `.media-width() { ...}` goes to the last file.

So to have the same structure I've created extra ***email-base-inline.scss*** and moved there inline stuff from ***email-base.scss***

and as in original magento in ***email-inline.less*** we have
```
@import 'source/_email-base.less'; // Contains primary email styles
@import 'source/_email-extend.less'; // Contains theme-specific adjustments to email styles
```
Then I also created ***_email-inline-extend.scss*** as I couldn't import here ***_email-extend.scss*** as there we will have content that shouldn't be inline.

I will add this explanation to the readme and general way of editing emails should cover Magento2 docs

Reply:
👍